lua/ttt2/libraries/spawn.lua

```diff
@@ -0,0 +1,290 @@
+---
+-- @desc A load of function handling player spawn
+-- @author Mineotopia
+
+local Vector = Vector
+local table = table
+local mathSin = math.sin
+local mathCos = math.cos
+local entsFindByClass = ents.FindByClass
+
+local spawnPointVariations = {Vector(0, 0, 0)}
+for i = 0, 360, 22.5 do
+	spawnPointVariations[#spawnPointVariations + 1] = Vector(mathCos(i), mathSin(i), 0)
+end
+
+local sizePlayer = Vector(32, 32, 72)
```

Comment:
I am not sure if you can just assume a player's size like that. Different playermodels (especially weird customized ones) can result in different player sizes.

Reply:
This is actually a great point, yes! It has been like this forever so I did not think about it at all. But yes, that's absolutely right!